### DISCUSSION TITLE
Unable to Pause Timer Using useIdleTimer Hook Inside Component

### DISCUSSION THREAD

**Discussion by FernandoVelcic** [SupremeTechnopriest/react-idle-timer#391] [General]
Hi! I have initialized the useIdleTimer in my React App.js and I would like to know if it's possible to access the pause and resume methods inside of a very deep component. I tried to do it by using the useIdleTimer hook again, but I couldn't pause the timer of my app. It seems like I'm recreating another instance of the idleTimer instead of reusing the already created in my App.js

Thanks!

**Comment by SupremeTechnopriest** [comment]
I think you want to use the provider. https://idletimer.dev/docs/api/idle-timer-provider

**Reply by FernandoVelcic** [reply]
Resolved, thanks.
